src/service-worker.js

```diff
@@ -0,0 +1,29 @@
+var cacheName = 'invoizr'
```

Comment:
I have already merge it, let see if it changes its cache name each time we create a new build, if it doesn't, it would be better to find another way. Thank you anyway :)